srcpkgs/libnice/template

```diff
@@ -1,23 +1,28 @@
 # Template file for 'libnice'
 pkgname=libnice
-version=0.1.17
-revision=2
+version=0.1.18
+revision=1
 build_style=meson
 build_helper="gir"
 configure_args="-Dcrypto-library=openssl -Dgtk_doc=disabled -Dtests=disabled
  -Dexamples=disabled -Dintrospection=$(vopt_if gir enabled disabled)"
-hostmakedepends="pkg-config glib-devel"
-makedepends="gstreamer1-devel libsoup-devel gupnp-igd-devel libressl-devel"
+hostmakedepends="pkg-config"
+makedepends="gstreamer1-devel libsoup-devel gupnp-igd-devel libressl-devel
+ glib-devel"
 short_desc="Implementation of the IETF's draft ICE (for P2P UDP data streams)"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.1-or-later"
 homepage="https://libnice.freedesktop.org/"
 distfiles="https://libnice.freedesktop.org/releases/libnice-${version}.tar.gz"
-checksum=1952a0dec58b5c9ccc3f25193df4e2d1244cb382cac611b71e25afcd7b069526
+checksum=5eabd25ba2b54e817699832826269241abaa1cf78f9b240d1435f936569273f4
 
 build_options="gir"
 build_options_default="gir"
 
+if [ "$CROSS_BUILD" ]; then
```

Comment:
Same as your other PR, just add `glib-devel` to `hostmakedepends` unconditionally, and see if `libglib-devel` is sufficient in `makedepends`. If not, keep the `glib-devel` in `makedepends` if needed.